### PR TITLE
Mjz/add request spec

### PR DESCRIPTION
**Pull request by mzagaja**
CI is currently failing because Travis doesn't have `settings.yml` with valid database credentials. Not sure what the best pattern is to surmount this yet.

### CODE REVIEW

**Comment by ericyoungberg** [comment]
Yeah, that's tricky. I'm going to have to give our friend Travis a look. I wonder if there is a way in the Travis dashboard to keep private files with our account that it can pull from to use when testing certain projects.

**Comment by iangmhill** [comment]
So, I think we should be committing the settings.yml and replacing the sensitive information with environment variables. Travis does allow env variables to be set as part of the settings for the repo.
screen shot 2018-03-23 at 8 43 50 am

File: app/geospatial_metadata.rb [hunk 1]
```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env ruby
+require 'active_record'
+require 'active_support/core_ext/hash'
+require 'yaml'
+require 'pry-byebug'
+require 'nokogiri'
+require 'rack'
+
+module GeospatialMetadata
```
Comment (iangmhill) [inline app/geospatial_metadata.rb]: I'm still learning Rails tests, so maybe the pattern in this file is very common, but could we add a quick comment for this test? I gather that we're testing metadata here, but it's unclear to me what part of the process we're actually trying to evaluate.

File: app/sync_geospatial_data_with_carto.rb [hunk 2]
```diff
@@ -4,19 +4,19 @@
...
 
 ActiveRecord::Base.establish_connection(
   adapter:  'postgresql',
-  host: @settings['host'],
-  port: @settings['port'],
-  database: @settings['gis_database'],
-  username: @settings['database_username'],
-  password: @settings['database_password'],
-  schema_search_path: @settings['gis_schema']
+  host: @settings['database']['host'],
```
Comment (iangmhill) [inline app/sync_geospatial_data_with_carto.rb]: I like how these settings are standardized now.

File: settings.yml.sample [hunk 3]
```diff
@@ -1,10 +0,0 @@
-host:
```
Comment (iangmhill) [inline settings.yml.sample]: Should this have been a `git mv` rather than a normal `mv`? Then we could save the commit history for the file.

Comment (mzagaja) [reply settings.yml.sample]: StackOverflow [suggests](https://stackoverflow.com/questions/2314652/is-it-possible-to-move-rename-files-in-git-and-maintain-their-history) that `git mv` wouldn't do this, and my empirical attempt to compare the two suggests the same. If you know the magic incantation here I'm happy to try it though.

File: app/geospatial_metadata.rb (hunk 1 above)
Comment (mzagaja) [reply app/geospatial_metadata.rb]: Are you referring to the test or the class? In general, the descriptions for the tests should be enumerated in the 'describe' and 'it' text for a test. You're right though that the class could maybe use a top line comment.

**Pull request merged**